nautobot/dcim/views.py

```diff
@@ -4239,6 +4243,9 @@ def get_extra_context(self, request, instance):
         if self.action == "retrieve" and instance:
             devices = Device.objects.restrict(request.user).filter(controller_managed_device_group__controller=instance)
             devices_table = tables.DeviceTable(devices)
+            devices_table.columns.show("controller_managed_device_group")
+            devices_table.columns.show("capabilities")
+            devices_table.sequence = ("name", "controller_managed_device_group", "capabilities", "...")
```

Comment:
Huh, does `...` work here? TIL.

Reply:
Yep! It works! Anything that is not defined in the sequence will be covered by the `"..."`. I feel like we use this somewhere else, but I had to double check the documentation.